addons/l10n_cl/views/report_invoice.xml

```diff
@@ -214,6 +214,9 @@
             </div>
         </xpath>
 
+        <!-- Disable the recap of tax totals in company currency at the bottom right of the invoice -->
```

Comment:
Can you also add the reason on the comment? Something like `... to prevent duplicate info since it's already present in our custom tax totals`

Reply:
Hello @yosuanicolaus. Thank you for the review. Added the additional reasoning in the comment and added the opw tag in the commit.